apps/admin/frontend/src/screens/settings_screen.tsx

```diff
@@ -10,16 +10,26 @@ import {
 import { AppContext } from '../contexts/app_context';
 import { NavigationScreen } from '../components/navigation_screen';
 import { FormatUsbButton } from '../components/format_usb_modal';
+import { logOut } from '../api';
 
 export function SettingsScreen(): JSX.Element {
   const { logger, usbDrive } = useContext(AppContext);
+  const logOutMutation = logOut.useMutation();
 
   return (
     <NavigationScreen title="Settings">
       <Prose maxWidth={false}>
         <h2>Current Date and Time</h2>
         <p>
-          <SetClockButton>
+          <SetClockButton
+            logOut={async () => {
+              try {
+                await logOutMutation.mutateAsync();
+              } catch {
+                // Handled by default query client error handling
+              }
```

Comment:
I know this pattern is clunky, but it's currently the best option we have for passing promisified react-query mutations to reusable components while also preventing unhandled promise rejections.

Touched on in [this presentation slide](https://docs.google.com/presentation/d/1qcohhkeavCVjTZAAhGbbB7sbnvDJLlNov3OTLijfcvs/edit#slide=id.g1f8414d696e_0_361) from a while back

Reply:
FWIW, I believe we're leaning toward `.mutate()` over `.mutateAsync()` - and if we need to vary UI based on mutation status, we can get that from `logOutMutation.[status|data|error]`. Similar discussion here: https://github.com/votingworks/vxsuite/pull/3573#discussion_r1229774807

That being said, I think @jonahkagan  might have the most up-to-date guidance here.

(Also, this is not important & non-blocking for this PR - just made me remember previous discussions)

Reply:
Ah yes you're totally right! Copied and pasted from my previous PR (https://github.com/votingworks/vxsuite/pull/3608), which benefitted from the ability to `await` the mutation that I was passing to the `SetClockButton`

Here, I can def use plain `.mutate` since I don't need to `await` it in `SetClockButton`

Reply:
https://github.com/votingworks/vxsuite/pull/3614/commits/7322f0a3badd6b74880556a7cf06cb4760da48eb